README.md

```diff
@@ -41,11 +44,11 @@ pipeline {
 
   stages {
     stage('Build') {
-      agent { label 'worker 1' }
```

Comment:
I don't understand the reason for this change? is it that the label doesn't atke space?

Reply:
No, just wanted it to look closer to the credential Id. Should I change it back?

Reply:
well. I think it keeps the PR more kosher and focused on what this PR really changes.

Reply:
Ok. Changed now 419c572db075a4ead6c1e58c95f059d0db253a95